pom.xml

```diff
@@ -3202,6 +3202,7 @@
                 <param>com.puppycrawl.tools.checkstyle.api.*</param>
                 <!-- 1% mutation in FullIdent -->
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheckTest</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheckTest</param>
```

Comment:
Should we add percentage here, as above?

Reply:
it is not necessary, but I added it.

Reply:
Resolved.